src/stest/tests/set-up-directory

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Is there any reason that all of the files in the `tests` directory are shell scripts, but none of them have an `.sh` extension? Most editors will autodetect this from the shebang, but not all.

Reply:
It's because they're executable.

I feel like it's conventional that an executable file has no extension. I suppose the fact that these are shell scripts can be considered an implementation detail -- they could equally be C binaries and the caller needn't know.

On the other hand, if these scripts were non-executable bash "source code" files that I expected to be loaded with `source` or `.`, I would definitely add a `.bash` extension.